Habitica/AndroidManifest.xml

```diff
@@ -21,7 +22,8 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        tools:replace="android:icon">
```

Comment:
It maintains the original icon. Needed because of a conflict with one of the libs. This doesn't affect the app in anyway.